.sublime-project

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
Better to use `http://editorconfig.org/` and try `https://code.visualstudio.com/` it is really cool

Reply:
Thanks.
I'm really into Atom almost for a half of year now.